python/source/flight.rst

```diff
@@ -752,4 +752,210 @@ Running the client script, you should see the server printing out information ab
 .. _IBM article: https://www.ibm.com/docs/en/arl/9.7?topic=certification-extracting-certificate-keys-from-pfx-file
 .. _Windows: https://docs.microsoft.com/en-us/dotnet/core/additional-tools/self-signed-certificates-guide
 .. _Arrow testing data repository: https://github.com/apache/arrow-testing/tree/master/data/flight
-.. _openssl: https://www.ibm.com/docs/en/api-connect/2018.x?topic=overview-generating-self-signed-certificate-using-openssl
\ No newline at end of file
+.. _openssl: https://www.ibm.com/docs/en/api-connect/2018.x?topic=overview-generating-self-signed-certificate-using-openssl
+
+Propagating OpenTelemetry Traces
+================================
+
+Distributed tracing with OpenTelemetry_ allows collecting call-level performance
+measurements across a Flight service. In order to correlate spans across a Flight
+client and server, trace context must be passed between the two. This can be passed
```

Comment:
Are we OK with using jargon without (briefly) defining it?

Reply:
I think so. My interpretation of the Cookbook is that it's mainly for sharing ready to use examples, not walking the reader through all the concepts. I think by the time the user is googling "arrow flight opentelemetry python" they know what these terms mean.

Reply:
OK, sounds good to me.